SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -142,23 +142,28 @@ bool CreateGlobalStateManager(Worker_SnapshotOutputStream* OutputStream)
 	return Worker_SnapshotOutputStream_WriteEntity(OutputStream, &GSM) != 0;
 }
 
-bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
+bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream, Worker_EntityId& NextAvailableEntityID)
```

Comment:
@Vatyx @mattyoung-improbable are we going to remove Placeholders altogether soon?

Reply:
Yes, to be honest I think we can remove it now. I think most of the new code in this PR should sit in it's own function and not touch placeholders.